R/net.mod.simnet.R

```diff
@@ -125,21 +130,32 @@ resim_nets <- function(dat, at) {
 #'
 edges_correct <- function(dat, at) {
 
-  if (dat$control$resimulate.network == TRUE) {
+  resimulate.network <- get_control(dat, "resimulate.network")
+  tergmLite <- get_control(dat, "tergmLite")
+  groups <- get_param(dat, "groups")
+  active <- get_attr(dat, "active")
 
-    if (dat$param$groups == 1) {
-      old.num <- dat$epi$num[at - 1]
-      new.num <- sum(dat$attr$active == 1)
+  if (resimulate.network == TRUE) {
+
+    if (groups == 1) {
+      index <- at-1
+      old.num <- get_epi(dat, "num", index)
+      new.num <- sum(active == 1)
       dat$nwparam[[1]]$coef.form[1] <- dat$nwparam[[1]]$coef.form[1] +
         log(old.num) -
         log(new.num)
     }
-    if (dat$param$groups == 2) {
-      group <- dat$attr$group
-      old.num.g1 <- dat$epi$num[at - 1]
-      old.num.g2 <- dat$epi$num.g2[at - 1]
-      new.num.g1 <- sum(dat$attr$active == 1 & group == 1)
-      new.num.g2 <- sum(dat$attr$active == 1 & group == 2)
+    if (groups == 2) {
+      index <- at-1
+      if (tergmLite == FALSE){
+        group <- idgroup(dat$nw)
+      } else {
```

Comment:
no need to use idgroup here even if tergmLite is FALSE. can always just lookup with get_attr. Right?